code/modules/mob/living/say.dm

```diff
@@ -267,102 +267,67 @@ proc/get_radio_key_from_channel(var/channel)
 
 	//Atmosphere calculations (speaker's side only, for now)
 	var/turf/T = get_turf(src)
+
 	if(T)
-		//Air is too thin to carry sound at all, contact speech only
+		//make sure the air can transmit speech - speaker's side
 		var/datum/gas_mixture/environment = T.return_air()
-		var/pressure = environment ? environment.return_pressure() : 0
+		var/pressure = (environment) ? environment.return_pressure() : 0
 		if(pressure < SOUND_MINIMUM_PRESSURE)
 			message_range = 1
 
-		//Air is nearing minimum levels, make text italics as a hint, and muffle sound
+		//sound distortion pressure, to help clue people in that the air is thin, even if it isn't a vacuum yet
 		if(pressure < ONE_ATMOSPHERE * 0.4)
-			italics = 1
-			sound_vol *= 0.5
+			italics = TRUE
+			sound_vol *= 0.5 //muffle the sound a bit, so it's like we're actually talking through contact
+
+		//DO NOT FUCKING CHANGE THIS TO GET_OBJ_OR_MOB_AND_BULLSHIT() -- Hugs and Kisses ~Ccomp

```

Comment:
I'd strongly recommend against including this change, hear() is kind of a hacky proc and there's a reason get_mobs_and_objs_in_view_fast is used, it's more efficient. Presumably CComp had something against it, but I have no idea why.

Reply:
Seconded. If you're unsure of the quality of the port, then you likely haven't done any tests to verify whether this is a performance improvement or not. Since that's the only real difference to be had from this change, if it doesn't improve performance I see no reason why to include it.